dev/source/docs/building-setup-windows10.rst

```diff
@@ -24,12 +24,12 @@ The official instructions are `here <https://docs.microsoft.com/en-us/windows/ws
     .. image:: ../images/build-on-windows10-wsl-install.png
         :target: ../_images/build-on-windows10-wsl-install.png
 
-#. From a web browser open `https://aka.ms/wslstore <https://aka.ms/wslstore>`_ which should open the Microsoft store to allow installing Ubuntu. Then Launch Ubuntu and fill in a username and login
+#. From a web browser open `https://aka.ms/wslstore <https://aka.ms/wslstore>`_ which should open the Microsoft store to allow installing and launching Ubuntu 22.04.  If the launch fails it may be necessary to open a command prompt (as administrator) and run "wsl --update" and "wsl --shutdown" before launching Ubuntu 22.04 again and then following the setup steps.
```

Comment:
I'm a bit surprised the order isn't `wsl --shutdown` then `wsl --update`. 

Also updates to wsl are now served through windows update itself.  (So I think that is how I had my client do an update at some point.)

Either way the `wsl --update` should achieve the same thing.